scripts/release.sh

```diff
@@ -53,6 +52,8 @@ git tag $new_tag
 git push origin $new_tag
 git push
 
+popd
```

Comment:
Without this, the `git add crates/utils/translations` was in the wrong location.

Reply:
So you execute the script from inside `scripts/` folder? Seems like it would break if you run it from project root, would be good if it worked from both places (or throws an error if its the wrong working dir).

Reply:
Yep I always run scripts from their directory.

Reply:
I always run them from the root path. Please add the following lines so that both ways work:

https://github.com/LemmyNet/lemmy/pull/3389/files#diff-52afcc694c0ff8ff253441f42ff44d6b4734fe30ae9070f2ba07a668a10e7baaR4-R6

Reply:
K done.